db/schema.rb

```diff
@@ -1,16 +1,15 @@
 # frozen_string_literal: true
 
-
-ActiveRecord::Schema.define(version: 20170810030748) do
-
-  create_table "members", force: :cascade do |t|
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.string "username"
-    t.string "firstname"
-    t.string "lastname"
-    t.string "email"
-    t.string "bio"
-    t.string "avatar"
+ActiveRecord::Schema
```

Comment:
We should probably exclude the schema file in the Rubocop config since this file is auto-generated and will update outside of our control. This could happen in the aforementioned Rubocop cleanup PR.

Reply:
Thanks!  The .rubocop.yml exclude does list db/**/* so I removed my rubocop exclusion to the schema version and the frozen_string_literal comment.  This PR was initially caught between the image PR and the Rubocop PR.   Hope we're okay going forward, but have noted to put such changes in separate PR.

Reply:
@veronicacannon all changes to this file can be excluded in this PR. The exclusion in `.rubocop.yml` is working as expected and Rubocop is passing as expected on `master`.